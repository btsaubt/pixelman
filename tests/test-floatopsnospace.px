def int main(){
	print_float(5.0-2.0);
	print_float(1.0+2.0);
	print_float(1.0*3.0);
	print_float(3.0/1.0);
}