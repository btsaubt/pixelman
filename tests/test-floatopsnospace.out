3.0000003.0000003.0000003.000000
